js/runner.go

```diff
@@ -54,6 +55,8 @@ type Runner struct {
 	BaseDialer net.Dialer
 	Resolver   *dnscache.Resolver
 	RPSLimit   *rate.Limiter
+
+	setupData interface{}
```

Comment:
We need to be able to extract this data, when running in cloud execution mode, and pass it to other nodes' `default` function. Maybe through the REST API? So, you could start k6 with something like `k6 cloud --run-setup-then-pause script.js`, extract the `setupData`, pass it to all nodes (`k6 run --setup-data "..."` or via REST API), and then unpause all nodes to being execution.

Reply:
It would be really nice if we could also supply `setup` as a path to a separate file (where this file exports `setup`, for example).
See [this comment](https://github.com/loadimpact/k6/issues/194#issuecomment-361584847) for background information.